Packs/Twitter/Integrations/Twitterv2/Twitterv2.py

```diff
@@ -0,0 +1,360 @@
+    users = include.get('users', [])
+    media = include.get('media', [])
+    next_token = response.get('meta', {}).get('next_token')
+    list_dict_response = []
+    for data_item in data:
+        author_id = data_item.get('author_id')
+        for user in users:
+            id = user.get('id')
+            if author_id == id:
+                dict_to_append = {
+                    'id': data_item.get('id', None),
+                    'text': data_item.get('text', None),
+                    'conversation_id': data_item.get('conversation_id', None),
+                    'created_at': data_item.get('created_at', ''),
+                    'edit_history_tweet_ids': data_item.get('edit_history_tweet_ids', None),
+                    'author': {'id': user.get('id', None),
+                               'description': user.get('description', None),
+                               'name': user.get('name', None),
+                               'verified': user.get('verified', None),
+                               'username': user.get('username', None),
+                               'created_at': user.get('created_at', None)},
+                    'public_metrics': data_item.get('public_metrics', {}),
+                    'media': []
+                }
+                attachments = data_item.get('attachments', {})
+                if attachments:
+                    media_to_append = []
+                    media_keys = attachments.get('media_keys', [])
+                    for media_key in media_keys:
+                        for media_item in media:
+                            media_key_attachments = media_item.get('media_key')
+                            if media_key == media_key_attachments:
+                                media_to_append.append({'url': media_item.get('url', None),
+                                                        'media_key': media_item.get('media_key', None),
+                                                        'alt_text': media_item.get('alt_text', None),
+                                                        'type': media_item.get('type', None)})
+                    dict_to_append['media'] = media_to_append
+                list_dict_response.append(remove_empty_elements(dict_to_append))
+    return list_dict_response, next_token
+
+
+def create_context_data_get_user(response: dict, pinned_tweets: str) -> list[dict]:
+    """ Gets raw response form Twitter API and extracts the relevent data.
+        The data matched by pinned_tweet_id == includes.tweets.id
+            Args:
+                response: dict - raw response form Twitter API.
+            Returns:
+                dict: context data.
+    """
+    include = response.get('includes', {})
+    data = response.get('data', {})
+    tweets = include.get('tweets')
+    list_dict_response = []
+    for data_item in data:
+        pinned_tweet_id = data_item.get('pinned_tweet_id')
+        dict_to_append = {'name': data_item.get('name', None),
```

Comment:
Why do we need all the Nones here?

Reply:
This enables to the function remove_empty_elements to remove it, if it is empty.

Reply:
Same as above